PHP Bitwise NOT Results in Wrong Class

### Short Summary
Code intel tried to look up class by name`~MyConstants`.


### Steps to Reproduce
Type the last colon in a file like this:
```php
<?php
class MyConstants
{
	const BITWISE1 = 0x00000001;
}
123 & ~MyConstants::
```


### Expected results
Autocompletion of `BITWISE1`.


### Actual results
Error:
```
[ERROR] koCodeIntel:     INFO: eval '~MyConstants' at test.php#8  <Trigger 'php-complete-static-members' at 80>
```

### Platform Information
Komodo Edit 11.1.2 Nightly
maxOS Mojave

--- a/src/udl/skel/PHP/pylib/lang_php.py
+++ b/src/udl/skel/PHP/pylib/lang_php.py
@@ -642,7 +642,7 @@ class PHPLangIntel(CitadelLangIntel, ParenStyleCalltipIntelMixin,
         WHITESPACE = tuple(" \t\n\r\v\f")
         EOL = tuple("\r\n")
         BLOCKCLOSES = tuple(")}]")
-        STOPOPS = tuple("({[,&$+=^|%/<;:->!.@?")
+        STOPOPS = tuple("({[,&$+=^|%/<;:->!.@?~")
         EXTRA_STOPOPS_PRECEDING_IDENT = BLOCKCLOSES # Might be others.
 
         #TODO: This style picking is a problem for the LangIntel move.

